ccd-definition/ComplexTypes/CareSupervision/JudicialMessage/JudicialMessage.json

```diff
@@ -15,6 +15,14 @@
     "ElementLabel": "Sent to",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "JudicialMessage",
+    "ListElementCode": "requestedBy",
+    "FieldType": "Text",
+    "ElementLabel": "Requested by",
+    "SecurityClassification": "Public"
+  },
   {
     "LiveFrom": "01/01/2017",
     "ID": "JudicialMessage",
```

Comment:
Is there a reason we are using a text field here for `dateSent` instead of the DateTime field?